############################################
# Configuration parameters for the Image Processing
############################################

ImageReader:
  image_folder: "/AppleDataCapture/shapefiles/"
